index.html

```diff
@@ -2528,10 +2533,11 @@ <h3>#xmlLang-root</h3>
         </section>
     </section> <!-- extensions-->
 
-  <section id="acknowledgements">
+  <section class="appendix">
       <h2>Acknowledgments</h2>
       The editors would like to thank XXX for their contributions to this specification.
   </section>
-
+  <section id="conformance">
+  </section>
```

Comment:
This is a breaking change - there's already a conformance section (§6).

Reply:
Ah, thank you for pointing this.
Yes, this was put by accident (and regression by section id="" edits), and let me fix as before.